<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PBearbeiten</class>
 <widget class="QWidget" name="PBearbeiten">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="lblUsername">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>90</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Username:</string>
   </property>
  </widget>
  <widget class="QLabel" name="berschhirft">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt; Dein Profil&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonProfil">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>380</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Speichern</string>
   </property>
  </widget>
  <widget class="QLabel" name="blackBackround">
   <property name="geometry">
    <rect>
     <x>567</x>
     <y>-3</y>
     <width>151</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="profilePicture">
   <property name="geometry">
    <rect>
     <x>581</x>
     <y>11</y>
     <width>121</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Username">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>130</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(241, 241, 241);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonExit">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>420</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Abbrechen </string>
   </property>
  </widget>
  <widget class="QLabel" name="lblemail_3">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>250</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Neues Passwort</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="email">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>200</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwortAlt">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>290</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwortNeu">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>280</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblemail_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>260</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Aktuelles Passwort</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblemail">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>170</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>e-Mail Adresse:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ProfilLoeschen">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>420</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Profil l√∂schen</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
